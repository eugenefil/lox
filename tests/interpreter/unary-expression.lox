-5;
--5;
!true;
!false;
!!false;
