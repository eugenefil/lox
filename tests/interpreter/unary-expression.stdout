-5
5
false
true
false
